src/ripple/app/ledger/LedgerProposal.h

```diff
@@ -55,15 +55,18 @@ class LedgerProposal
         std::uint32_t proposeSeq,
         uint256 const& propose,
         NetClock::time_point closeTime,
+        NetClock::time_point now,
         PublicKey const& publicKey,
         NodeID const& nodeID,
+        Slice const& signature,
```

Comment:
There's really on advantage to passing it by value and there's a possible disaddvange, a new Slice may actually get constructed (though probably won't).